packages/react-log-viewer/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@patternfly/react-log-viewer",
+  "version": "4.0.0",
+  "description": "Terminal renderer",
+  "main": "dist/js/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/js/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public",
+    "tag": "prerelease"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/patternfly/patternfly-react.git"
+  },
+  "keywords": [
+    "react",
+    "patternfly",
+    "code-editor"
+  ],
+  "author": "Red Hat",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/patternfly/patternfly-react/issues"
+  },
+  "homepage": "https://github.com/patternfly/patternfly-react/tree/master/packages/react-log-viewer#readme",
+  "scripts": {
+    "clean": "rimraf dist"
+  },
+  "dependencies": {
+    "@patternfly/react-core": "^4.106.4",
+    "@patternfly/react-icons": "^4.9.10",
+    "@patternfly/react-styles": "^4.9.5",
+    "@types/react-window": "^1.8.1",
+    "classnames": "^2.2.6",
+    "memoize-one": "^5.1.0",
+    "react-window": "^1.8.6",
+    "react-window-infinite-loader": "^1.0.7",
+    "tslib": "1.13.0"
+  },
+  "peerDependencies": {
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
+    "react-monaco-editor": "^0.41.2"
```

Comment:
Remove `react-monaco-editor` from peer dependencies. Remember to run `yarn install` to update the `yarn.lock` after changing `package.json`.